src/resolve/resolved-conf.c

```diff
@@ -414,3 +414,61 @@ int manager_parse_config_file(Manager *m) {
         return 0;
 
 }
+
+int config_parse_refuse_record_types(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Manager *m = ASSERT_PTR(userdata);
+
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        if (isempty(rvalue)) {
+                data = mfree(data);
+                return 1;
+        }
+
+        r = free_and_strdup_warn(data, empty_to_null(rvalue));
```

Comment:
What? Please drop. This leaks the copied string.

Reply:
Drop everything related to `data`  and use `rvalue` instead so this code is no longer necessary.